moveit_core/trajectory_processing/test/test_ruckig_traj_smoothing.cpp

```diff
@@ -106,7 +106,7 @@ TEST_F(RuckigTests, basic_trajectory_with_custom_limits)
 TEST_F(RuckigTests, trajectory_duration)
 {
   // Compare against the OJET online trajectory generator: https://www.trajectorygenerator.com/ojet-online/
-  const double ideal_duration = 0.21025;
+  const double ideal_duration = 0.310;
```

Comment:
I was looking at the wrong joint limits -- it is reading from the robot model, not joint_limits.yaml for this test.